optimize/client/src/components/Home/AlertList.js

```diff
@@ -157,8 +150,8 @@ const AlertList = ({collection, readOnly}) => {
         rows={
           !isLoading &&
           alerts.map((alert) => {
-            const {id, name, webhook, emails, reportId, threshold, thresholdOperator} = alert;
-            const inactive = webhook && emails?.length === 0 && !webhooks?.includes(webhook);
+            const {id, name, emails, reportId, threshold, thresholdOperator} = alert;
+            const inactive = emails?.length === 0;
```

Comment:
I think we do not need the inactive warning anymore because it seems to be that this will not be possible anymore.
We introduced this in the context of the webhooks. See:
https://jira.camunda.com/browse/OPT-3237